src/fping.c

```diff
@@ -1323,6 +1332,16 @@ void finish()
     if (stats_flag)
         print_global_stats();
 
+    if (reachable_flag) {
+        if ((num_hosts-num_unreachable >= reachable)) {
+            printf("Number of reachable hosts: %d\n", num_hosts-num_unreachable);
+            exit(0);
+        } else {
+            printf("<%d hosts are reachable\n", reachable);
```

Comment:
Here too:

```suggestion
            printf("Not enough hosts reachable (required: %d, reachable: %d)\n", reachable, num_hosts-num_unreachable);
```

Reply:
Made the required modification in both test cases and the code.